app/controllers/api/applications_controller.rb

```diff
@@ -19,6 +19,8 @@ class Api::ApplicationsController < FrontendController
 
   sublayout 'api/service'
 
+  helper_method :presenter
```

Comment:
This feels weird, especially because the presenter is only for the `new` action.

Reply:
It's also used in `create` (see PR description).